docs/reference/api/docker_remote_api.md

```diff
@@ -163,6 +163,7 @@ This section lists each version from latest to oldest.  Each listing includes a
 * Every API response now includes a `Docker-Experimental` header specifying if experimental features are enabled (value can be `true` or `false`).
 * The `hostConfig` option now accepts the fields `CpuRealtimePeriod` and `CpuRtRuntime` to allocate cpu runtime to rt tasks when `CONFIG_RT_GROUP_SCHED` is enabled in the kernel.
 * The `SecurityOptions` field within the `GET /info` response now includes `userns` if user namespaces are enabled in the daemon.
+* The `HostConfig` field now includes `NanoCPUs` that represents CPU CFS (Completely Fair Scheduler) quota in units of 10<sup>-9</sup> CPU shares.
```

Comment:
Thanks @crosbymichael @mlaventure @stevvooe. The wording has been changed to:

The `HostConfig` field now includes `NanoCPUs` that represents CPU quota
in units of 10<sup>-9</sup> CPUs.